### PR TITLE
Add ability to compute a velocity layer

### PR DESCRIPTION
**Pull request by forrestfwilliams**

### CODE REVIEW

File: src/opera_disp_tms/generate_sw_disp_tile.py [hunk 1]
```diff
@@ -69,39 +72,70 @@ def find_needed_granules(frame_ids: Iterable[int], begin_date: datetime, end_dat
         frame_ids: The frame ids to generate the tile for
         begin_date: Start of secondary date search range to generate tile for
         end_date: End of secondary date search range to generate tile for
+        strategy: Selection strategy for granules within search range ("minmax" or "all")
```
Comment (jacquelynsmale) [inline src/opera_disp_tms/generate_sw_disp_tile.py:75]: Is the strategy for the search range of the dates? 
```suggestion
        strategy: Selection strategy for granules within search date range ("max", "minmax" or "all")
```

File: src/opera_disp_tms/generate_sw_disp_tile.py [hunk 2]
```diff
@@ -69,39 +72,70 @@ def find_needed_granules(frame_ids: Iterable[int], begin_date: datetime, end_dat
...
         else:
-            granule = max(granules, key=lambda x: x.secondary_date)
-            needed_granules.append(granule)
+            raise ValueError(f'Invalid strategy: {strategy}. Must be "seacant" or "all".')
+
     return needed_granules
 
 
-def create_product_name(metadata_name: str, begin_date: datetime, end_date: datetime) -> str:
+def create_product_name(
```
Comment (jacquelynsmale) [inline src/opera_disp_tms/generate_sw_disp_tile.py:104]: Will this be used in another script do you think? Would this type of function be good to start a `util` file?

Comment (forrestfwilliams) [reply src/opera_disp_tms/generate_sw_disp_tile.py:104]: Yep, you're probably makes more sense in utils.py

Comment (forrestfwilliams) [reply src/opera_disp_tms/generate_sw_disp_tile.py]: Done!

File: scripts/make_cal_meta_tiles.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import argparse
 from pathlib import Path
 
-from opera_disp_tms.generate_frame_tile import create_product_name, create_tile_for_bbox
+from opera_disp_tms.generate_metadata_tile import create_product_name, create_tile_for_bbox
```
[the remark this answers is not in the document]
Comment (jacquelynsmale) [reply scripts/make_cal_meta_tiles.py:4]: Also need to update line 14 product_name = Path(create_product_name(['metadata'], orbit_direction, bbox) + '.tif')

**Review by jacquelynsmale — APPROVED** [review]

**Pull request merged**